server/src/services/database.service.js

```diff
@@ -37,7 +37,7 @@ export default class DatabaseServiceProvider {
           await Promise.allSettled([this.TARGET.TableService.destroy(tableName)]);
           await this.TARGET.TableService.create(constructSchema(Table));
 
-          const params = { Limit: 100 };
+          const params = {};
```

Comment:
@maskeynihal , This params is related to the no. of items to restore per table at a time. It's not related to the no. of Tables to restore.

```
await this.SOURCE.ItemService.fetch(OPERATIONS.SCAN, tableName, params);
```